test/WebSites/SimpleWebSite/Startup.cs

```diff
@@ -22,7 +22,6 @@ public void ConfigureServices(IServiceCollection services)
                 .AddAuthorization()
                 .AddFormatterMappings(m => m.SetMediaTypeMappingForFormat("js", new MediaTypeHeaderValue("application/json")))
                 .AddJsonFormatters(j => j.Formatting = Formatting.Indented);
-            services.AddSingleton<UrlEncoder, UrlTestEncoder>();
```

Comment:
Leave this alone.

Reply:
Same reason as others. We can add it if we want in future.